chaoscenter/web/src/views/CreateProject/CreateProject.tsx

```diff
@@ -0,0 +1,117 @@
+    createProjectMutation(
+      {
+        body: {
+          projectName: values.projectName,
+          description: values.projectDescription,
+          tags: values.tags
+        }
+      },
+      {
+        onSuccess: () => {
+          handleClose();
+        }
+      }
+    );
+  }
+
+  const initialValues: CreateProjectFormProps = {
+    projectName: '',
+    projectDescription: '',
+    tags: []
+  };
+
+  return (
+    <Layout.Vertical padding="medium" style={{ gap: '1rem' }}>
+      <Layout.Horizontal flex={{ alignItems: 'center', justifyContent: 'space-between' }}>
+        <Text font={{ variation: FontVariation.H4 }}>{'Create Project'}</Text>
+      </Layout.Horizontal>
+      <Container>
+        <Formik<CreateProjectFormProps>
+          initialValues={initialValues}
+          onSubmit={values => handleSubmit(values)}
+          validationSchema={Yup.object().shape({
+            projectName: Yup.string().required(getString('usernameIsRequired')),
+            projectDescription: Yup.string()
+          })}
+        >
+          {formikProps => {
+            return (
+              <Form style={{ height: '100%' }}>
+                <Layout.Vertical style={{ gap: '2rem' }}>
+                  <Container>
+                    <FormInput.Text
+                      name="projectName"
+                      placeholder={getString('enterYourName')}
+                      label={<Text font={{ variation: FontVariation.FORM_LABEL }}>{getString('name')}</Text>}
+                    />
+                    <FormInput.Text
+                      name="projectDescription"
+                      placeholder={getString('projectDescription')}
+                      label={
+                        <Text font={{ variation: FontVariation.FORM_LABEL }}>{getString('projectDescription')}</Text>
+                      }
+                    />
+                    <FormInput.KVTagInput
+                      name="tags"
+                      label="Tags"
+                      isArray={true}
+                      tooltipProps={{
+                        dataTooltipId: 'tagInputId'
+                      }}
+                    />
+                  </Container>
+                  <Layout.Horizontal style={{ gap: '1rem' }}>
+                    <Button
+                      type="submit"
+                      variation={ButtonVariation.PRIMARY}
+                      text={createProjectMutationLoading ? <Icon name="loading" size={16} /> : getString('confirm')}
+                      loading={createProjectMutationLoading || formikProps.isSubmitting}
+                      disabled={createProjectMutationLoading || Object.keys(formikProps.errors).length > 0}
+                      style={{ minWidth: '90px' }}
+                    />
+                    <Button
+                      variation={ButtonVariation.TERTIARY}
+                      text={getString('cancel')}
+                      onClick={() => handleClose()}
```

Comment:
Not necessary but this can be shortened.
```suggestion
                      onClick={handleClose}
```